fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -46,11 +47,30 @@ def self.install_wwdr_certificate
       Dir.chdir('/tmp') do
         url = 'https://developer.apple.com/certificationauthority/AppleWWDRCA.cer'
         filename = File.basename(url)
-        `curl -O #{url} && security import #{filename} -k login.keychain`
+        keychain = wwdr_keychain
+        keychain.prepend("-k ") unless keychain.empty?
+        `curl -O #{url} && security import #{filename} #{keychain}`
         UI.user_error!("Could not install WWDR certificate") unless $?.success?
       end
     end
 
+    def self.wwdr_keychain
+      priority = [
+        "security list-keychains -d user",
+        "security default-keychain -d user"
+      ]
+      for command in priority
```

Comment:
Explicitly using the `for` loop construct to create the variables in the function scope.